addons/aircraft/functions/fnc_droneSetWaypoint.sqf

```diff
@@ -36,7 +36,30 @@ _pos set [
 
 // [_group] call CBA_fnc_clearWaypoints;
 _waypoint = _group addWaypoint [_pos, 0];
-_waypoint setWaypointType _type;
+if (_type == "FOLLOW" && {["CAManBase", "LandVehicle", "Ship"] findIf {_target isKindOf _x} != -1}) then {
+    _waypoint setWaypointType "HOLD";
+    //_waypoint waypointAttachVehicle _target;
```

Comment:
Is this needed or not? If not, delete.

Reply:
True that, the comment is not needed anymore, forgot to remove it.